bangazonreports/templates/products/expensiveproducts.html

```diff
@@ -6,7 +6,7 @@
         <title>Bangazon Reports</title>
     </head>
     <body>
-        <h1>Favorite Sellers by Customer</h1>
+        <h1>Expensive Products</h1>
```

Comment:
Did you mean IN-expensive products here?

Reply:
haha yes. whoops.

Reply:
oh actually, no I meant Expensive products. I realized that I hadn't corrected it from 'Favorite Sellers by Customer' to 'Expensive Products' when I pushed that commit on Tuesday